src/Tribe/Integrations/WPML/WPML.php

```diff
@@ -66,6 +66,10 @@ protected function hook_filters() {
 		$language_switcher = Tribe__Events__Integrations__WPML__Language_Switcher::instance();
 		add_filter( 'icl_ls_languages', array( $language_switcher, 'filter_icl_ls_languages' ), 5 );
 
+		$meta = Tribe__Events__Integrations__WPML__Meta::instance();
+		add_filter( 'get_post_metadata', array( $meta, 'translate_post_id' ), 10, 3 );
+		add_filter( 'pre_get_posts', array( $meta, 'include_all_languages' ) );
```

Comment:
when we start using `tribe_singleton` we can replace the `array` to `tribe_callback`